### ISSUE TITLE
Search capability on Participant Tables

### ISSUE THREAD

**Issue by afolarin** [RADAR-base/ManagementPortal#236]
The number of rows of the MP when large will be impractical to identify a given participant from their IDs, I would suggest it worth having the ability to search the fields of the table (at least the main participant view).

**Comment by nivemaham** [comment]
Indeed. This could be a starting point. https://www.jhipster.tech/using-elasticsearch/

**Comment by blootsvoets** [comment]
Perhaps basic sql searches could also do? Aren't the tables small enough that they can handle a search of this size?

**Comment by afolarin** [comment]
SQL index should be adequate

**Comment by afolarin** [comment]
I would think Elasticsearch (though very cool) is a fairly big hammer to crack a nut? Also adding another non-trivial dependency?
Also there are some security considerations (e.g. API restriction) vis-a-vis Elasticsearch that might need to be considered (if not using XPACK (thought the ODES https://opendistro.github.io/for-elasticsearch/ is relatively new)?

**Comment by afolarin** [comment]
Additionally looks like the sources tends to grow faster... to the extent that the tabulation UI is overflowing (even an ultrawide screen :smile: ) 
![image](https://user-images.githubusercontent.com/2489685/81161773-00cb4d00-8f84-11ea-88e8-9c918af8b7e1.png)

Generally, search or at least filtering functionality would be essential for larger projects.
